framework/build.sh

```diff
@@ -7,25 +7,27 @@ cd build
 
```

Comment:
Will this file not be used by the linux builds too? Do we need to add checks to see what platform we are on for things like the OSX-specific stuff or does cmake just ignore it?

Reply:
It will just get ignored. It should be safe to have a common `build.sh`